ngraph/core/src/op/lstm_cell.cpp

```diff
@@ -145,6 +145,21 @@ bool ngraph::op::v0::LSTMCell::visit_attributes(AttributeVisitor& visitor)
 void op::v0::LSTMCell::validate_and_infer_types()
 {
     NGRAPH_OP_SCOPE(v0_LSTMCell_validate_and_infer_types);
+
+    // There should be 7 inputs, if no, it's possible the op can be fixed by
+    // generating default ones for input 6 and 7 (bias input, peepholes)
+
+    // missing both bias_input and peepholes
+    if (get_input_size() == 5)
+    {
+        set_argument(5, get_default_bias_input());
```

Comment:
We usually set default inputs inside operation c-tor. Is it applicable for LSTMCell?

Reply:
Yes, usually this would not be needed, but for conversion we create all of the ops using default c-tor, then set the inputs separately. This is fine for most cases but as you can see for the other constructors of LSTMCell we generate one or two default inputs, depending of how many inputs we have, this can't be the case for the default constructor as at that point we're still waiting for the inputs to be set.